be/sidedish-be/src/main/java/com/codesquad/sidedish/utils/SampleDataFactory.java

```diff
@@ -0,0 +1,68 @@
+package com.codesquad.sidedish.utils;
+
+import com.codesquad.sidedish.web.sidedish.DTO.DetailDTO;
+import com.codesquad.sidedish.web.sidedish.DTO.ItemDTO;
+import com.codesquad.sidedish.web.sidedish.DTO.SidedishDTO;
+import com.fasterxml.jackson.core.JsonParseException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import org.springframework.core.io.ClassPathResource;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class SampleDataFactory {
+
+    private SampleDataFactory() {
+
+    }
```

Comment:
```suggestion
    private SampleDataFactory() {
    }
```

빈 body는 비워주는 것이 흔히 사용하는 컨벤션입니다.